core/src/test/resources/sql/sub-query.iq

```diff
@@ -3544,4 +3544,33 @@ FROM UNNEST(ARRAY['a', 'b']) AS a(attidentity);
 
 !ok
 
+# Test case for [CALCITE-5680] Wrong plan for multiple IN sub-queries with only literal operands
```

Comment:
@JiajunBernoulli We only add non-correlated version test case here, not including a correlated one.
This is because the correlated one will throw exception now, see [CALCITE-5390](https://issues.apache.org/jira/browse/CALCITE-5390), after we fix CALCITE-5390, the correlated one should be added here.

Reply:
Ok, I get it.